docs/preprocess.rst

```diff
@@ -125,7 +135,44 @@ This pipeline can be run through the functions saved in ``site_pipeline``. Each
 entry in "process_pipe" key will be used to generate a Preprocess module based
 on the name it is registered to. These entries will then be run in order through
 the processing pipe. The ``process`` function is always run before the
-``calc_and_save`` function for each module.
+``calc_and_save`` function for each module. The ``plot`` function can be run after
+``calc_and_save`` when ``plot: True`` for a module that supports it. Current modules
+with plotting: ``DetBiasFlags``, ``EstimateHWPSS``, ``SSOFootprint``.
```

Comment:
Now every time you add a new module you need to remember to update this list here...I might just leave out that last sentence.

Reply:
seconded